content/tutorials/build-apps/build-a-desktop-wallet-in-js.md

```diff
@@ -0,0 +1,243 @@
+---
+parent: build-apps.html
+targets:
+  - en
+  - ja # TODO: translate this page
+blurb: Build a graphical desktop wallet for the XRPL using Javascript.
+---
+# Build a Desktop Wallet in Javascript
+<!-- STYLE_OVERRIDE: wallet -->
+
+This tutorial demonstrates how to build a desktop wallet for the XRP Ledger using the Javascript programming language and various libraries. This application can be used as a starting point for building a more complete and powerful application, as a reference point for building comparable apps, or as a learning experience to better understand how to integrate XRP Ledger functionality into a larger project.
+
+## Prerequisites
+
+To implement this tutorial you should have a basic understanding of JavaScript and Node.js. You should also have a basic idea about XRP Ledger. For more information, visit the [XRP Ledger Dev Portal](https://xrpl.org) and the [XRPL Learning Portal](https://learn.xrpl.org/) for videos, libraries, and other resources.
+
+Follow the steps below to get started:
+
+1. [Node.js](https://nodejs.org/en/download/) (v10.15.3 or higher)
+2. Install [Yarn](https://yarnpkg.com/en/docs/install) (v1.17.3 or higher) or [NPM](https://www.npmjs.com/get-npm) (v6.4.1 or higher)
+3. Add your Seed, Client, and specify testnet/mainnet in .env file. Example .env file is provided in the source code below.
+4. Run `yarn install` or `npm install` to install dependencies
+5. Start the app with `yarn dev` or `npm dev`
```

Comment:
These aren't pre-requisites because you'll walk them through doing this as part of the tutorial. We can just get rid of this for now.
```suggestion
```